manifests/load-watcher-deployment.yaml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
One thing to note here is that we would like to deploy load-watcher container in the same pod as the scheduler when using it as a service, to minimise HTTP call latency. I have added instructions to deploy it as a service in the same pod as scheduler [here](https://medium.com/paypal-engineering/real-load-aware-scheduling-in-kubernetes-with-trimaran-a8efe14d51e2). Users would use what we provide so I suggest we either add the above instructions or remove this file for a future enhancement.

Reply:
Let's just remove it for now.